### PR TITLE
IDVA5-1482 New links to clear session when verifying another client

### PR DESCRIPTION
**Pull request by rnesbitt-ch**
**Short description outlining key changes/additions**

- Added new section to confirmation view with 2 links as per prototype: verify another client and authorised agent account.
- New controller with GET method to clear the USER_DATA and CHECK_YOUR_ANSWERS_FLAG session variables to allow user to go through verify journey again.
- Redirecting user to correct url.
- Unit test class for new controller.

**JIRA Ticket Number**

[IDVA5-1482](https://companieshouse.atlassian.net/browse/IDVA5-1482)

**Type of change**
 
- [ ] Bug fix
- [X]  New feature
- [ ]  Breaking change
- [ ]  Infrastructure change

**Pull request checklist**

- [X]  I have added unit tests for new code that I have added
- [ ]  I have added/updated functional tests where appropriate
- [X]  The code follows our [coding standards](https://github.com/companieshouse/styleguides/blob/master/java.md)

**An exhaustive list of peer review checks can be read** [here](https://github.com/companieshouse/styleguides/blob/master/java_review.md#developer-actions-prior-to-code-commitreview-started)

[IDVA5-1482]: https://companieshouse.atlassian.net/browse/IDVA5-1482?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

File: src/views/confirmation/confirmation.njk [hunk 1]
```diff
@@ -39,6 +39,7 @@
   {% set howIdentityDocsChecked = i18n.option2Confirmation %}
 {% endif %}
 
+<form action="" id="confirmationForm" method="POST">
```
Comment (ttingle-ch) [inline src/views/confirmation/confirmation.njk:42]: You do not need a form tag as there are no input fields for the user to enter data

Comment (rnesbitt-ch) [reply src/views/confirmation/confirmation.njk]: Thanks Troy I've removed that now

File: src/views/confirmation/confirmation.njk [hunk 2]
```diff
@@ -107,4 +108,14 @@
       }
     ]
   }) }}
+
+  <h2 class="govuk-heading-m">{{ i18n.whatToDoNext }}</h2>
+  <p class="govuk-body">
+    <a href="/tell-companies-house-you-have-verified-someones-identity/confirmation-redirect?id=verify-service-link" class="govuk-link" id="verify-service-link"">{{ i18n.linkToVerifyService }}</a>
```
Comment (cbroadley-ch) [inline src/views/confirmation/confirmation.njk]: I think you may have an extra closing " here under the id name field: `verify-service-link`

File: locales/en/confirmation.json
Comment (cbroadley-ch) [inline locales/en/confirmation.json:1]: You will need to add some Welsh translation place holder text in the cy > confirmation.json file to match this

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

File: locales/en/confirmation.json
Comment (rnesbitt-ch) [reply locales/en/confirmation.json:1]: Have added those in now thanks

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Review by swalke-ch — APPROVED** [review]
LGTM

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Review by ttingle-ch — APPROVED** [review]
LGTM

**Review by cbroadley-ch — APPROVED** [review]
LGTM

**Pull request merged**